src/clusterfuzz/_internal/metrics/monitoring_metrics.py

```diff
@@ -175,6 +175,23 @@
     ],
 )
 
+UTASK_DURATION_SECS = monitor.CumulativeDistributionMetric(
```

Comment:
Shouldn't this have subtask in the name?

Reply:
Fair point! The existing e2e metric does not have it, so I went for consistency. I can switch both over if you'd rather.

Reply:
Done.